engo_sdl.go

```diff
@@ -32,7 +34,8 @@ var (
 	Gl           *gl.Context
 	sdlGLContext sdl.GLContext
 
-	scale = float32(1)
+	scale       = float32(1)
+	shouldClose bool
```

Comment:
Since we are now using a channel for close notification in our other renderers, we should follow the same pattern here.

Reply:
Yeah, but that leads to deadlock since the check to exit is in the update loop due to sdl's poll events. I think there's a callback I can use instead though. I'll look.

Reply:
Fixed both SDL and GLFW to use the channel exclusively ^_^!